stanford-earth/SE3_BLT
# EARTH-156: Adds su-sws/stanford_news and enables it.

# READY FOR REVIEW

# Summary
- First pass at a stanford_news type for D8.

# Needed By (Date)
- Before the July release

# Urgency
- Normal

# Steps to Test

1. Check out this branch
2. Run `composer update --prefer-source`
2b. Check out EARTH-156 of the matson theme
3. Clear caches
4. Create a new news item 
5. Review display

Please also have a review of the 8.x-1.x branch code in stanford_news. There is no PR for that so it will require manual review of the new thing.

# Associated Issues and/or People
- EARTH-156

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
